zipline/finance/performance/period.py

```diff
@@ -193,15 +227,14 @@ def position_tracker(self, obj):
         # we only calculate perf once we inject PositionTracker
         self.calculate_performance()
 
-    def rollover(self):
-        self.starting_value = self.ending_value
-        self.starting_exposure = self.ending_exposure
-        self.starting_cash = self.ending_cash
-        self.period_cash_flow = 0.0
-        self.pnl = 0.0
-        self.processed_transactions = {}
-        self.orders_by_modified = {}
-        self.orders_by_id = OrderedDict()
+    def rollover(self, capital_change=0):
```

Comment:
I agree that `rollover` in it's entirety doesn't need to be called again in `process_capital_changes`. All we really need to do is increment the starting cash, which can be done as you suggested